platform-includes/configuration/integrations/javascript.node.mdx

```diff
@@ -0,0 +1,33 @@
+### Integrations
+
+|                                                           | **Auto Enabled** | **Errors** | **Performance** | **Additional Context** |
+| --------------------------------------------------------- | :--------------: | :--------: | :-------------: | :--------------------: |
+| [`consoleIntegration`](./console)                         |        ✓         |            |                 |           ✓            |
+| [`contextLinesIntegration`](./contextlines)               |        ✓         |     ✓      |                 |                        |
+| [`dedupeIntegration`](./dedupe)                           |        ✓         |     ✓      |                 |                        |
+| [`functionToStringIntegration`](./functiontostring)       |        ✓         |            |                 |                        |
+| [`graphqlIntegration`](./graphql)                         |        ✓         |            |        ✓        |                        |
+| [`httpIntegration`](./http)                               |        ✓         |            |        ✓        |           ✓            |
```

Comment:
l: I think it makes sense to mark `httpIntegration` also for errors since it gives us request isolation and basic scope transactionName updates

```suggestion
| [`httpIntegration`](./http)                               |        ✓         |     ✓      |        ✓        |           ✓            |
```

Reply:
0ce87b32cbc055be196f5febc3077eb1dc5fe615